flake.nix

```diff
@@ -3,7 +3,7 @@
 
   inputs = {
     typelevel-nix.url = "github:typelevel/typelevel-nix";
-    nixpkgs.follows = "typelevel-nix/nixpkgs";
+    # nixpkgs.follows = "typelevel-nix/nixpkgs";
```

Comment:
Temurin was recently added to nixpkgs.  We can roll this back after the next typelevel-nix update, which should kick off tonight.